cmd/manager/main.go

```diff
@@ -175,6 +177,15 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Setup operator specific metrics
+	muoMetrics := prometheus.NewRegistry()
+	muoMetrics.MustRegister(
+		prometheus.NewGoCollector(),
+		prometheus.NewProcessCollector(prometheus.ProcessCollectorOpts{
+			Namespace: "muo",
```

Comment:
Think it's worth being explicit here, `managed-upgrade-operator`.

Reply:
so i originally had the MetricsNamespace of "managed_upgrade" as that is the service the metrics are reporting. However for this collector, its more about the operator underneath. The reason I used an acronym was due to reviewing https://github.com/OpenObservability/OpenMetrics/blob/feb021c735e9d61ee5779f6e581b07f098304f9b/specification/OpenMetrics.txt#L1835-L1840

Happy to stand corrected tho.

Reply:
That's cool, I wasn't entirely sure what the implications where as to what happens with a namespace like that.
My main reason for noting it was that "muo" means nothing to anyone except us, so it might inhibit its potential usefulness to others who see it and have no context for what it means.
I kind of like `managed_upgrade_`.